core/src/main/java/com/linecorp/armeria/server/FallbackService.java

```diff
@@ -35,9 +35,9 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exc
         final RoutingContext routingCtx = ctx.routingContext();
         final HttpStatusException cause = getStatusException(routingCtx);
```

Comment:
How about also fixing `getStatusException` not creating `HttpStatusException` and use `HttpStatusException` only if `routingCtx.deferredStatusException()` returns it?

Reply:
Sounds good.